Tests/MapboxNavigationTests/CarPlayManagerSpec.swift

```diff
@@ -33,7 +33,10 @@ class CarPlayManagerSpec: QuickSpec {
         
         afterEach {
             CarPlayMapViewController.unswizzleMethods()
-            carPlayManager = nil
+            // FIXME: There is a deadlock in `NavigationEventsManager` while calling its deinit method
+            // (most likely `TelemetryService` or `EventsService` cannot be deallocated correctly).
+            // This is the reasons tests stuck and eventually fail.
+            // carPlayManager = nil
```

Comment:
@chezzdev, please take a look at this one. It seems that deadlock happens in either `TelemetryService` or `EventsService`. Their allocation process is also strange (I mean `getOrCreate` method). I've explicitly call `telemetryService = nil` and deadlock occurs right after that (you can pause debugger to see that).